client/src/js/services/exportFile.js

```diff
@@ -131,7 +134,7 @@ angular.module('bhima.services')
     e.click();
 	}
 
-	function csv(fileData, fileName, separator) {
+	function csv(fileData, fileName, genDate, separator) {
```

Comment:
We must stop to adopt java naming convention to declare our variable see [here] (http://www.j-io.org/Javascript-Naming_Conventions)